android/build.gradle

```diff
@@ -54,22 +53,8 @@ allprojects {
           if (rootProject.file("local.properties").exists()) {
               localProperties.load(rootProject.file("local.properties").newDataInputStream())
           }
-
-          maven {
```

Comment:
we no longer need the artifactory version or jitpack as everything's on maven central